pyproject.toml

```
@@ -1,18 +1,25 @@
 [build-system]
-requires = ["setuptools>=61", "cffi>=1.3.0", "requests", "setuptools-scm", 'asn1crypto']
-build-backend = "setuptools.build_meta"
+requires = [
+    'scikit-build-core>=0.9.0',
+    'hatchling',
+    'setuptools>=42.0.0',
+    'cffi>=1.3.0',
+    'pkgconfig',
```

I can confirm that this actually does nothing but the following makes it work for me:

```suggestion
    'pypkgconf; sys_platform == "win32"',
```

---

Could you try with `pkg-config`? It is a conda package, `pypkgconf` is not. I could work on making it so, but I am tied-up with some difficult common Lisp recipe atm

---

I think because that one is a mere interface for an existing binary on PATH whereas the one that works for me provides that binary directly (although called `pkgconf`, I guess tools support that name as well) since I don't have that installed and I'm quite hesitant to use the binary from SourceForge that was built 11 years ago. Maybe we can remove that from CI as well?

If Conda doesn't have that yet for some reason do you want me to merge, release, and then you can work on that afterward?

---

You can always ignore this dependency on Conda Forge and just use the other one. Build dependencies don’t always match on conda, it’s a different ecosystem.